dist/version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "21.1.2"
+  "version": "21.1.3"
```

Comment:
Why is this 21.1.3, but the version in package.json is 21.1.4?

Reply:
^^ let me know when it's been corrected and I'll approve :)

Reply:
@DLarson-Oddball @ryan-mcneil fixed